releases/8.0.0.md

```diff
@@ -278,6 +278,10 @@ JavaScript:
             ...
         ]
         ```
+    - Add django.contrib.admin to the existing post addition to INSTALLED_APPS:  
```

Comment:
Does this work as intended without removing the first reference to the admin app at the top?

Reply:
added a note to that effect